bin/sender.py

```diff
@@ -36,28 +36,34 @@ def main():
     """Set up connection, send all messages and quit."""
     ver = "SSM %s.%s.%s" % __version__
     default_conf_location = '/etc/apel/sender.cfg'
-    op = OptionParser(description=__doc__, version=ver)
-    op.add_option('-c', '--config',
-                  help=('location of config file, '
-                        'default path: ' + default_conf_location),
-                  default=default_conf_location)
-    op.add_option('-l', '--log_config',
-                  help='DEPRECATED - location of logging config file (optional)',
-                  default=None)
-
-    options, unused_args = op.parse_args()
+    arg_parser = ArgumentParser(description=__doc__)
+
+    arg_parser.add_argument('-c', '--config',
+                            help=('location of config file, default path: '
+                                  '%s' % default_conf_location),
```

Comment:
```suggestion
                            help='location of config file, default path: '
                                  '%s' % default_conf_location,
```

This is the only one using parentheses so should tidy them up as they're not necessary in this case.